REFERENCE.md

```diff
@@ -1,432 +0,0 @@
-<p align="center">
```

Comment:
@kaxil it still references using single brackets in aql.transform

Reply:
That might be a typo -- can we fix that instead.

I remember @sunank200 had recently updated it when changing append/merge interface.

The biggest benefit of using Sphinx + Our example DAGs is that we can avoid this kind of things and instead of harc-coding the code one example, we can extract it from the working code. Example: check this PR - https://github.com/astronomer/astronomer-providers/pull/473